Libraries/PushNotificationIOS/PushNotificationIOS.js

```diff
@@ -165,6 +165,35 @@ class PushNotificationIOS {
   }
 
   /**
+   * Remove all delivered notifications from Notification Center
+   */
+  static removeAllDeliveredNotifications(): void {
+    RCTPushNotificationManager.removeAllDeliveredNotifications();
+  }
+
+  /**
+   * Provides you with a list of the app’s notifications that are still displayed in Notification Center
+   *
+   * @param callback Function which receive an array of delivered notifications
+   */
+  static getDeliveredNotifications(callback: (notifications: [Object]) => void): void {
```

Comment:
You can type Object here to give more information about the fields you transform https://flowtype.org/docs/objects.html